crates/goose-cli/src/main.rs

```diff
@@ -1,26 +1,16 @@
 use anyhow::Result;
```

Comment:
not sure what exactly changed here to cause this, but I wasn't able to run `goose run -t "tell me a joke"`. it errors out with `error: the following required arguments were not provided:
  --instructions <FILE>` 

works in the latest release version though

Reply:
scratch that, was running the wrong build!